x/nft/simapp/app.go

```diff
@@ -0,0 +1,600 @@
+package simapp
+
+import (
```

Comment:
thanks for adding this, any chance you can get rid of all modules except what is needed? this will help with reducing the changes needed in this file in the future.

Reply:
This PR should probably be closed because someone is doing the same thing